src/Sulu/Bundle/PageBundle/Teaser/TeaserContentType.php

```diff
@@ -37,38 +37,13 @@
  */
 class TeaserContentType extends SimpleContentType implements PreResolvableContentTypeInterface, PropertyMetadataMapperInterface
 {
-    /**
-     * @var TeaserProviderPoolInterface
-     */
-    private $teaserProviderPool;
-
-    /**
-     * @var TeaserManagerInterface
-     */
-    private $teaserManager;
-
-    /**
-     * @var ReferenceStorePoolInterface
-     */
-    private $referenceStorePool;
-
-    /**
-     * @var PropertyMetadataMinMaxValueResolver|null
-     */
-    private $propertyMetadataMinMaxValueResolver;
-
     public function __construct(
-        TeaserProviderPoolInterface $providerPool,
-        TeaserManagerInterface $teaserManager,
-        ReferenceStorePoolInterface $referenceStorePool,
-        ?PropertyMetadataMinMaxValueResolver $propertyMetadataMinMaxValueResolver = null
+        private TeaserProviderPoolInterface $teaserProviderPool,
```

Comment:
We're changing the parameter name to match the name of the property. Is that a BC break?

Reply:
Fair enough.

Reply:
Then it should be good to merge.